daff/README.md

```diff
@@ -0,0 +1,224 @@
+are in all 3 files when using compound primary keys.
+
+**Result:** Daff is able to capture row maping of rows that are 3 files when compound primary keys are in use.
+
+**Relevant Issues:** 1, 2
+
+### Case 5
+
+**Prerequisite:** Assume that the preprocessing has been completed. This preprocessing removes columns that the local
+file doesn’t care about from the ancestor and remote files. Assume local and remote files independently add
+3 columns that turn out to contain the same information. Assume there are 2 columns that exist in all 3 (ancestor,
+local remote) files. In the remote file assume all compound primary key cells are edited.
+
+**Goal:** Test if compound R-L columns can accurately capture row mapping for remote-local rows. Also test the
+minimum number of compound primary keys needed to accruately capture row mapping and what errors take place when all
+compound primary key cells are updated.
+
+**Result:** Daff is not able to capture row maping for rows that appear in all 3 (ancestor, local, remote) files when
+all compound primary key cells are edited in the remote file. Daff is not able to capture row maping for rows that appear
+in only remote and local files. This has to do with the fact that daff doesn't show column/row merge conflicts. The daff
+library only reports cell level merge conflicts. The minimum number of compound primary keys needed are 2.
+
+**Relevant Issues:** 1, 2, 3
+
+### Case 6
+
+**Prerequisite:** Assume that the preprocessing has been completed. This preprocessing removes columns that the local
+file doesn’t care about from the ancestor and remote files. Assume local and remote files independently add
+a column that turns out to contain the same information. Assume there are 2 columns that exist in all 3 (ancestor,
+local remote) files. In the local file assume the all cell in the A-L column and all compound primary key cells are
+edited.
+
+**Goal:** Unlike the remote file no errors occur when all compound primary key cells are edited in the local file. This
+case identifies an error that occurs after editing the cells of the local file mentioned in the **prerequisite**.
+
+**Result:** A duplicate row for rows found in all 3 files is added in the diff output.
+
+**Relevant Issues:** 1, 2, 4
+
+### Case 7
+
+**Prerequisite:** Assume that the preprocessing has been completed. This preprocessing removes columns that the local
+file doesn’t care about from the ancestor and remote files. Assume local and remote files independently add
+a column that turns out to contain the same information. Assume there are 2 columns that exist in all 3 (ancestor,
+local remote) files. In the local file assume cells that are found in only ancestor and local files are edited.
+
+**Goal:** Highlight a case were row issue 1 does not occur.
+
+**Result:** Row issue 1 does not occur. The output is what the WeCount accessibility map would expect besides the
+pending issues 2 and column issue 1.
```

Comment:
what is the "column issue 1"?

Reply:
It is daff always removing the A-L column.

Reply:
In this line I am clarifying that even though the removal of ancestor-local rows are not taking place, the removal of the A-L column is still a problem.